src/date_input.jsx

```diff
@@ -95,24 +95,12 @@ var DateInput = React.createClass({
   },
 
   render () {
-    const {
```

Comment:
I had troubles to build the code locally and I had to remove this piece of code. I can revert if it's needed.